gluten-ut/src/test/scala/io/glutenproject/utils/velox/VeloxNotSupport.scala

```diff
@@ -48,6 +63,8 @@ object VeloxNotSupport extends NotSupport {
     simpleClassName[NondeterministicSuite],
     simpleClassName[RandomSuite],
     simpleClassName[ArithmeticExpressionSuite],
-    simpleClassName[ConditionalExpressionSuite]
+    simpleClassName[ConditionalExpressionSuite],
+    simpleClassName[GlutenDataFrameAggregateSuite]
```

Comment:
If test name starts with `GlutenTestConstants.GLUTEN_TEST`, test will always run, but in intellij, you cannot specify one test to run, always run all the test in the suite, do not know why.

It will run the test in `GlutenDataFrameAggregateSuite`, not `DataFrameAggregateSuite`

Reply:
So somehow it works but may not work in the expected way.

This line from our code:

https://github.com/oap-project/gluten/blob/36bb6434ffdc24bdb5fe5de386176927d3ef5da3/gluten-ut/src/test/scala/org/apache/spark/sql/GlutenTestsBaseTrait.scala#L34

Shows that we use `GlutenDataFrameAggregateSuite`'s super class name `DataFrameAggregateSuite` to do some sort of matching. 

Thus I think it doesn't matter we put line 67 here or not.

I think it's the test framework itself that needs improvements to make developer know what happens exactly.